apps/address_registry/tests/views/test_sodra_views.py

```diff
@@ -1,55 +1,166 @@
-from datetime import date
-
 import pytest
-from spyne import Fault
-from spyne.client.django import DjangoTestClient
-
-from apps.address_registry.views.sodra_views import skola_sodrai
+from django.test import Client
+from lxml import etree
 
 
 @pytest.fixture
-def client() -> DjangoTestClient:
-    return DjangoTestClient("/api/v1/sodra/skola-sodrai/", skola_sodrai.app)
+def client() -> Client:
+    return Client()
 
 
-def _get_request_data(asm_kodas: int = 11111111111) -> dict:
-    return {
-        "SkolaSodrai": {
-            "asm_kodas": asm_kodas,
-            "vardas": "Vardenis",
-            "pavarde": "Pavardenis",
-            "gim_data": date(2023, 1, 1),
-            "sds": "1",
-            "sdn": 1,
-            "klausejo_kodas": 1,
-            "tikslas": 1,
-        }
-    }
+def _get_element_text(root_node, element_name: str, namespaces: dict) -> str | None:
+    el = root_node.find(element_name, namespaces=namespaces)
+    return el.text if el is not None else None
```

Comment:
Could we use the full names here?

```suggestion
    element = root_node.find(element_name, namespaces=namespaces)
    return element.text if element is not None else None
```